### DISCUSSION TITLE
Images not displaying in Bagisto admin panel, Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10198] [Q&A]
Images are not being shown in the Bagisto admin panel. The browser console shows the following error:
` Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID`

![Screenshot (551)](https://github.com/user-attachments/assets/ed2e31c8-c185-4915-a051-ac3707ee9f78)

Additionally, after trying to upload an image, it also doesn't appear on the front end.

![Screenshot (554)](https://github.com/user-attachments/assets/633e50db-81d5-4314-887e-8b3c95d1ed8a)

**Environment:**

Hosting: Local (XAMPP)
OS: Windows 10
Web Server: Apache
PHP Version: 8.2.12

Could you please advise on how to resolve this?

**Comment by devansh-webkul** [comment]
The error seems to suggest a potential issue with your SSL certificate.
